### ISSUE TITLE
Obtaining different results from Crude Monte Carlo Simulation and Importance Sampling

### ISSUE THREAD

**Issue by FahriBaran** [pystra/pystra#36]
I am using Pystra to perform a structural reliability analysis of the correlated frame (Example 3 in the article) investigated in the study of Blatman, G., & Sudret, B. (2010). An adaptive algorithm to build up sparse polynomial chaos expansions for stochastic finite element analysis. Probabilistic Engineering Mechanics, 25(2), 183–197. https://doi.org/10.1016/j.probengmech.2009.10.003

The calculated probability of failure of the frame - for 6.1 cm threshold displacement - is equal to 0.000348 for Crude Monte Carlo Simulation - by using 1149000 samples with 0.04999 coefficient of variation for the results -, however, the calculated probability of failure is equal to 0.000228 when the Importance Sampling method - by using 2000 samples with 0.048998 coefficient of variation for the results - is used. 
What might be the reason for these differences in results?

**Comment by ccaprani** [comment]
The MCS estimate compares with the results in [https://doi.org/10.1016/j.probengmech.2006.05.004](https://doi.org/10.1016/j.probengmech.2006.05.004), but for a limit of 0.2 inches, not 6.1cm (?). The number of IS samples is quite low though. 

If you discover an issue with the algorithms in pystra, please raise an issue, but I'll close this otherwise.

**Comment by FahriBaran** [comment]
The limit state function is actually 0.2ft in the imperial unit system which is approximately equal to 6.1 cm in the SI system. However, the main issue is that the algorithm stops when the CoV of the results is lower than 0.05 which might be the reason for the early stopping with 2000 samples. When I tried to increase the number of samples manually up to 1e4 by using options.samples command, it still gives the same result for IS. I further increased the number of samples to 1e5 but in that case, the CoV results are equal to 0.23 yielding a probability of failure approximately equal to the one obtained from CMCS. However, the CoV is suspiciously high. 

In the literature, this example seems to have two different solutions with little differences in their inputs for example the correlation matrix changes - some researchers correlate loads ro=0.95 - or the distribution type of the loads can be Gumbel Max depending on the article at hand. In my study, I am exactly following the inputs given in https://doi.org/10.1016/j.probengmech.2009.10.003. In that study, the reference beta is taken as equal to 3.51 (approximately equal to p_f=0.000224) for the 6 cm displacement threshold (given in Table 8). 

To sum up, if I increase the number of samples in IS, I get high CoV. Otherwise, the algorithm stops when CoV is less than 0.05. In the second case, I am ending up with a low number of samples. Is there any way to overcome this problem? Thank you for your time and interest by the way.
